### ISSUE TITLE
Programmatically load next/previous video in a playlist

### ISSUE THREAD

**Issue by x3rocode** [PierfrancescoSoffritti/android-youtube-player#887]
# Feature Request

### Describe your idea
Thanks for the addition of new features that using playlist.

1. Load next/previous vedio when use playlist 
 - I can't found next or prev button when using playlist ( This feature is already provided by YouTube API V3. ). or get current video number, and the total number of videos to changing vedio.
 
2. Sign with google to use YouTube Premium (remove ads, play private viedos)  I found solution !!!!!!! [here](https://github.com/PierfrancescoSoffritti/android-youtube-player/issues/845)
 - I don't know it's possible. for using YouTube Premium sign in Google and choose account before playing video to playing private viedos (or playlists) and remove ads. I tried get an access token using OAuth2.0 to use the permission. However, it is not compatible with the 'YouTubePlayerView' library provided by youtube.  
  
3. Get total time of current viedo.

### Expected behavior and realization

1. 
```java
private lateinit var player: YouTubePlayer
                     :
youTubePlayerView.initialize(API_KEY, this@MainActivity)

playbtn.setOnClickListener {
            player.play()
        }
pausebtn.setOnClickListener {
            player.pause()
        }

prevbtn.setOnClickListener {
           if(player.hasPrevious){
                    player.previous()
            }
        }
nextbtn.setOnClickListener {
           if(player.hasNext){
                    player.next()
            }
        }
                :

override fun onInitializationSuccess(
        p0: YouTubePlayer.Provider?,
        p1: YouTubePlayer?,
        p2: Boolean
    ) {
        /** add listeners to YouTubePlayer instance **/
        p1?.setPlayerStateChangeListener(playerStateChangeListener);
        p1?.setPlaybackEventListener(playbackEventListener);

        if (!p2) {
            p1?.cuePlaylist(PlayList_ID);
        }
        player = p1!!
    }
```
2. 
![image](https://user-images.githubusercontent.com/51151970/190533766-b4f34c2c-3500-4415-8434-676756202d4b.png)
![image](https://user-images.githubusercontent.com/51151970/190533801-dc9aa6f9-fa3e-4aac-8307-2671600c2f68.png)

**Comment by PierfrancescoSoffritti** [comment]
This will be available in version 12.1.0 which releases sometimes this week.

**Comment by MuhammadAshikurRahman** [comment]
i want after end of video next video will play automatic
